### DISCUSSION TITLE
Enhancement needed: Add SRT automatically

### DISCUSSION THREAD

**Discussion by quebecman** [RandomEngy/VidCoder#844] [Ideas]
Unfortunately, there is missing the NAME of the SUBTITLE. Would be nice to get the one that is already entered down in the Custom Name Track.

Maybe only a checkmark: Use Custom Track Name.

It works as advertize. Nice to have m8. It's way faster like this than having to REMUX all the time!

**Comment by RandomEngy** [comment]
This kind of thing belongs as an issue, not under discussions. But, noted.
